components/Workspaces/WorkspaceWelcomeModal.tsx

```diff
@@ -73,12 +79,17 @@ export default function WorkspaceWelcomeModal({ isOpen, onClose }: WorkspaceWelc
     <Dialog open={isOpen}>
       <DialogContent autoStyle={false} onEscapeKeyDown={onClose} onPointerDownOutside={onClose}>
         <Card className="sm:w-96 md:w-max max-w-xl text-center flex flex-col items-center justify-between gap-4 !p-8">
-          {/* Using !p-8 for now as the Card component has explicit padding of p-3. We can revisit. */}
-          <img
-            src="/assets/workspace_overview.png"
-            alt="Workspace screenshot from documentation"
+          <iframe
+            width="100%"
+            height="300px"
```

Comment:
The aspect ratio is off on smaller screens. Use the `aspect-video` Tailwind class in conjunction with the height.

![CleanShot 2024-05-21 at 11 12 45](https://github.com/open-sauced/app/assets/833231/9e9b18f7-60d4-4b86-8cd0-af087ebbb362)

Reply:
I removed the `height` and added the `aspect-video` class which seems to be responsive across the different screens.